.taskcluster.yml

```diff
@@ -250,7 +250,40 @@ tasks:
               - project-relman
               - bugbug
           metadata:
-            name: bugbug update hooks
-            description: bugbug update hooks
+            name: bugbug update data hook
+            description: bugbug update data hook
+            owner: ${user}@users.noreply.github.com
+            source: ${repository}/raw/${head_rev}/.taskcluster.yml
+
+      - $if: 'tasks_for == "github-push" && head_branch[:10] == "refs/tags/"'
+        then:
+          taskId: {$eval: as_slugid("update_hook_check_pipeline")}
+          dependencies:
+            - {$eval: as_slugid("docker_push")}
+          scopes:
+            - hooks:modify-hook:project-relman/bugbug
+            - assume:hook-id:project-relman/bugbug
```

Comment:
You'll need to use a different hook, or one of the two is going to overwrite the other.